pr-temp/Where-To-Put-Documentation.md

```diff
@@ -0,0 +1,117 @@
+As with all large projects, the Fluid Framework requires a sizable amount of documentation.
+That documentation lives in a number of different locations / systems.
+It can sometimes be difficult to find the documentation you need.
+Similarly, it can sometimes be hard to determine where new documentation should live.
+
+This document aims to give simple guidance to make our documentation systems easier to navigate, and to ensure we have consistent policy with regards to where different forms of documentation should go.
+
+## Documentation locations
+
+The following are locations where Fluid Framework documentation can be found.
+They are not presented in any particular order.
+
+- [fluidframework.com](https://fluidframework.com/)
+  - This is our customer-facing website.
+    It is the public face of our product and is home to high-level overviews, samples, etc. to get users up-and-running on the framework.
+- [Github Wiki](https://github.com/microsoft/FluidFramework/wiki)
+  - This is the contributor-facing wiki for the [fluid-framework](https://github.com/microsoft/FluidFramework) repository.
```

Comment:
Might be nice to clarify here that the GitHub wiki is mainly for contributing guidelines since the other locations have brief summaries of their main purposes.

Reply:
Good call!